proposals/lip-0051.md

```diff
@@ -2448,7 +2448,9 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
   * The `supplySubstore` array must be in ascending order of `tokenID`.
   * Across all elements of the `escrowSubstore` array, the `tokenID` must be unique (it appears at most once).
   * The `escrowSubstore` array must be in ascending order of `tokenID`.
-  * The `supportedTokensSubstore` array must be in lexicographical order of `chainID` and the `chainID` should be of length `CHAIN_ID_LENGTH` or `0`. For each entry of this array, the `supportedTokenIDs` array should be in lexicographic order.
+  * The `supportedTokensSubstore` array must adhere to one of the following two mutually exclusive conditions:
```

Comment:
This is in case the array is non-empty. An empty array is also valid input. So I would say that if the array is non-empty, then it must satisfy those constraints. 

Minor comment, although a like the "mutually exclusive" phrase, might be confusing for devs, to it would be good to use either ... or ... . But not strong opinion here.

Reply:
Hmm, what is your suggestion, how to write it to be more clear?